lorien/UI/Toolbar.gd

```diff
@@ -165,3 +165,34 @@ func _change_active_tool_button(btn: TextureButton) -> void:
 # -------------------------------------------------------------------------------------------------
 func get_brush_color_button() -> Control:
 	return _color_button
+
+# -------------------------------------------------------------------------------------------------
+func _on_active_project_changed(previous_project: Project, current_project: Project) -> void:
```

Comment:
This should be a public method (ie. no leading underscore). Maybe just call it `change_active_project`